spec/lib/audit/catalog_to_moab_existence_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../../../lib/audit/catalog_to_moab_existence.rb'
+require_relative '../../load_fixtures_helper.rb'
+
+RSpec.describe CatalogToMoabExistence do
+  include_context 'fixture moabs in db'
+  describe '.whole_db' do
+    it 'test_of_load_fixtures_helper (eventually will be testing code in .whole_db)' do
+      described_class.whole_db
+    end
+  end
+end
```

Comment:
I'm perfectly happy to leave this as a bare-bones PR, but could you at least write down some comments here on what we should eventually be testing?

Reply:
again - part of ticket #108.  This is strictly meant here as "does my test fixture loading work?"